Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/clipboard/CodeBrowserClipboardProvider.java

```diff
@@ -382,6 +390,21 @@ else if (delta > 0) {
 		return createStringTransferable(StringUtils.join(strings, "\n"));
 	}
 
+	private Transferable copyByteSourceOffset(TaskMonitor monitor) {
+		AddressSetView addrs = getSelectedAddresses();
+		Memory currentMemory = currentProgram.getMemory();
+		List<String> strings = new ArrayList<>();
+		AddressIterator ranges = addrs.getAddresses(true);
+		while (ranges.hasNext() && !monitor.isCancelled()) {
+			Address nextAddress = ranges.next();
+			AddressSourceInfo addressSourceInfo = currentMemory.getAddressSourceInfo(nextAddress);
+			if (addressSourceInfo != null) {
+				strings.add(String.format("%x", addressSourceInfo.getFileOffset()));
```

Comment:
When the address is not backed by a file, `addressSourceInfo.getFileOffset()` will return `-1`, and `ffffffffffffffff` will get copied.  This seems undesirable.  Should probably copy either `-1` or something like `<NO_OFFSET>` or something similar.